src/ring_theory/integral_closure.lean

```diff
@@ -357,6 +357,50 @@ begin
   exact hp',
 end
 
+lemma is_integral_quotient_of_is_integral {I : ideal A} (hRS : ∀ (x : A), is_integral R x) :
```

Comment:
`(hRS : ∀ (x : A), is_integral R x)`
I think it is high time that we define what it means for a ring hom (or algebra) to be integral. Doesn't have to be this PR. But we shouldn't wait too long with it.

Reply:
I can make a PR for this change after this one gets merged, since I'm still working on other things using this stuff (don't want to balloon this PR to much bigger).